content/docs/1.3.0/deploy/install/_index.md

```diff
@@ -84,6 +84,11 @@ If your Kubernetes cluster was provisioned by Rancher v2.0.7+ or later, the Moun
 
 If MountPropagation is disabled, Base Image feature will be disabled.
 
+### Root and Privileged Permission
+
+Longhorn uses `Bidirectional` mount propagation design, therefore it is allowed only in privileged containers.
+Besides, Longhorn uses `nsenter` command to execute commands on the host. The `nsenter` command rely on privileged containers and `hostPID: true`.
```

Comment:
Could we identify what components use bidirectional mount or `nsenter`, and then briefly explain what cases/purposes?

Reply:
I add some wording to explain the purpose. Please check if it's good to understand or not.